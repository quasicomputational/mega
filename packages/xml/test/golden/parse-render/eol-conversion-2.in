<foo>
hello
what?&#xd;okay</foo>