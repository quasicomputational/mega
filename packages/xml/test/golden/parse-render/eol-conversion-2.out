<foo>

hello
what?&#xD;okay</foo>